hal/shared/platform_ncp.h

```diff
@@ -62,7 +66,11 @@ enum PlatformNCPIdentifier {
     PLATFORM_NCP_BROADCOM_BCM9WCDUSI09 = PLATFORM_NCP_IDENTIFIER(PLATFORM_NCP_MANUFACTURER_BROADCOM, 1),
     PLATFORM_NCP_BROADCOM_BCM9WCDUSI14 = PLATFORM_NCP_IDENTIFIER(PLATFORM_NCP_MANUFACTURER_BROADCOM, 2),
     PLATFORM_NCP_REALTEK_RTL872X = PLATFORM_NCP_IDENTIFIER(PLATFORM_NCP_MANUFACTURER_REALTEK, 1)
```

Comment:
I realized all existing P2s have been provisioned with an NCPID of `A0`, which unfortunately doesnt match this (`PLATFORM_NCP_REALTEK_RTL872X` should be `A1` right?). 
I think it should be changed to the following if we want to identify P2s as they are now:
```
PLATFORM_NCP_REALTEK_RTL872X = PLATFORM_NCP_IDENTIFIER(PLATFORM_NCP_MANUFACTURER_REALTEK, 0)
```

Reply:
Let's fix this then.